### DISCUSSION TITLE
Reset back to default settings

### DISCUSSION THREAD

**Discussion by kachidk** [RyotaUshio/obsidian-pdf-plus#277] [Q&A]
The option to reset back to default settings would be a great addition.

**Comment by RyotaUshio** [comment]
The easiest way is to uninstall PDF++ and reinstall it. This resets all settings to default. I will add "restore default" option for each setting as well in the future.

**Reply by RyotaUshio** [reply]
The next release [0.40.12](https://github.com/RyotaUshio/obsidian-pdf-plus/releases/tag/0.40.12) will include the following:

- "Restore default settings" command for restoring the default values of all options
- Each setting item in the PDF++ setting pane will have "Restore default value of this setting" context menu item.
